### PR TITLE
Cargo sensor

### PR DESCRIPTION
**Pull request by jojobean23**
Addition of code for a cargo sensor

### CODE REVIEW

File: src/main/java/org/usfirst/frc/team319/robot/commands/Elevator_Commands/toggleElevatorFloorSolenoid.java [hunk 1]
```diff
@@ -25,12 +25,7 @@ protected void initialize() {
   // Called repeatedly when this Command is scheduled to run
   @Override
   protected void execute() {
-    if(Robot.elevator.isElevatorFloorSolenoidExtended()) {
```
Comment (tytremblay) [inline src/main/java/org/usfirst/frc/team319/robot/commands/Elevator_Commands/toggleElevatorFloorSolenoid.java]: we deleted the toggle logic?

File: src/main/java/org/usfirst/frc/team319/robot/subsystems/Carriage.java [hunk 2]
```diff
@@ -17,40 +19,37 @@
...
   @Override
   public void periodic() {
+    SmartDashboard.putBoolean("Ball Detected", this.ballDetected());
   }
 
-  private double getCurrentPosition() {
-    return this.passThroughLead.getSelectedSensorPosition();
-  }
-
+  /*
```
Comment (tytremblay) [inline src/main/java/org/usfirst/frc/team319/robot/subsystems/Carriage.java]: we should delete this instead of commenting it

Comment (jojobean23) [reply src/main/java/org/usfirst/frc/team319/robot/subsystems/Carriage.java]: fixed

File: src/main/java/org/usfirst/frc/team319/robot/commands/Elevator_Commands/toggleElevatorFloorSolenoid.java (hunk 1 above)
Comment (jojobean23) [reply src/main/java/org/usfirst/frc/team319/robot/commands/Elevator_Commands/toggleElevatorFloorSolenoid.java]: fixed

**Pull request merged**